### DISCUSSION TITLE
Apache Project Issue Disabled

### DISCUSSION THREAD

**Discussion by dtonic-yj** [community/community#31693] [Other Feature Feedback, Questions, & Ideas] [answered]
# How do you raise an issue in GitHub Project with the issue tab disabled?

I'm trying to raise issue for a Apache Project ([Phoenix-Connectors](https://github.com/apache/phoenix-connectors) Project to be exact)

I tested this project (Phoenix-Connectors) and found some issues and questions.
It looks like this project hasn't been released for quite a while and developing seems to be still in progress (there's new commit 16 days ago)
So it's hard to find information about the project.

That is why I want to raise the issue directly to Contributor
But It seems like there 's no issue tab here..

Is there be any way to raise an issue in GitHub Project where the issue tab is disabled?
Or is there a site where I can usually get help in this case?

**Comment by airtower-luna** [comment, marked as answer]
That depends on the project. Read the README, check their website. For Apache projects you usually have to use either their own issue tracker, or a mailing list.

**Reply by dtonic-yj** [reply]
It really has its own issue tracking system,,, I couldn't find it initially,,,

Thank you so much !
